report-summary-merged-prs

```diff
@@ -976,6 +979,23 @@ def find_dup_dict_result(comparisons):
             })
 
 
+def find_profiling_results(test_field, comparisons, architecture, magic_command, results_function=find_one_test_results):
```

Comment:
no need to add this fucntion

Reply:
OK. Let me try this using the default function. There are two files that need to be found before the links are created.

Reply:
But find_general_test_results do not add the architecture field.

Reply:
just set  results_function=your_callback_fucntion for https://github.com/cms-sw/cms-bot/blob/master/report-summary-merged-prs#L979 and then your your_callback_fucntion can set architecture and workflow number . Avoid hard coding the workflow too.